src/pTSL.py

```diff
@@ -59,14 +99,11 @@ def get_projections(self, string):
             new_projections.append((proj, (1 - prob) * val))
         return new_projections
 
-    '''
-        read the corpus file into data and output the conditional probabilities fore each input
-        NOTE: unused for now. Maybe useful when implementing GUI?
-    '''
     def val_corpus(self, corpus_file):

```

Comment:
Can we just remove this method? The GUI can probably just use whatever workflow the program currently uses.